libs.autobuild

```diff
@@ -2,7 +2,54 @@
 
 cmake_package 'slam/threed_odometry'
 cmake_package 'gui/pose3d_editor'
+
 ruby_package 'control/ruby_sdformat'
+cmake_package "drivers/glfw" do |pkg|
+    pkg.define "BUILD_SHARED_LIBS", "ON"
+end
+
+#Needed for kinect, the os packages does not (yet) work 06.15
+Autoproj.add_osdeps_overrides 'glfw', :package => 'drivers/glfw'
+remove_from_default 'drivers/glfw'
+
+import_package "drivers/glew"  do |pkg|
+    pkg.depends_on "glfw"
+    pkg.depends_on "libturbojpeg"
+    pkg.post_install do
+        Autoproj.env_set_path "GLEW_DEST", Autobuild.prefix
```

Comment:
should be pkg.prefix, not Autobuild.prefix. Never assume that all packages have the same prefix.